app/templates/passwords/reset-password.html

```diff
@@ -50,24 +55,26 @@ <h1 class="saturn">Reset your password</h1>
     </ul>
 
     <fieldset id="reset-details" class="js-new-password-group">
-        {% if errorType != None %}
+
+        {% if errorType.password %}
```

Comment:
Agreed, suggest confirming the requirements with the RAD team and either removing the javascript as a defect or meet their defined requirement as an iteration.